src/components/events/SeatSwapMarketplace.tsx

```
@@ -0,0 +1,336 @@
+
+interface SeatSwapMarketplaceProps {
+  eventId: string;
+  user: User | null;
+}
+
+export function SeatSwapMarketplace({ eventId, user }: SeatSwapMarketplaceProps) {
+  const supabase = createClient();
+  const [selectedTargetSeatId, setSelectedTargetSeatId] = useState<string>("");
+
+  // 1. Fetch event seating layout and seats
+  const { data: seatingData, isLoading: isSeatingLoading, refetch: refetchSeats } = useQuery({
```

ESLint

[error] Replace `·data:·seatingData,·isLoading:·isSeatingLoading,·refetch:·refetchSeats` with `⏎····data:·seatingData,⏎····isLoading:·isSeatingLoading,⏎····refetch:·refetchSeats,⏎·`

(prettier/prettier)